setup.py

```diff
@@ -28,14 +28,10 @@
         "numpy",
         "rich",
         "rtree",  # Used by sumolib
-        "filelock",
-        "lz4",
-        "networkx",
-        "opencv-python",
         "pandas",
         "psutil",
         "visdom",
-        "pybullet",
+        "pybullet==3.0.6",
```

Comment:
Why was this needed?

Reply:
`python 3.8` updates `pybullet` which uses  the latest version of `numpy 1.20.3` which is not supported by `tensorflow`. So this is to ensure that pybullet is not updated during the setup.